cluster/gce/windows/configure.ps1

```diff
@@ -105,6 +105,11 @@ try {
   Create-Directories
   Download-HelperScripts
 
+  FetchAndImport-ModuleFromMetadata `
+      'install-logging-agent-psm1' `
```

Comment:
@mtaufen  this will add a new script. I think we've discussed this and no new script should be added.

Reply:
I've updated the PR to not add a new file and disrupt the "release machinery."